dotnet/src/Microsoft.AutoGen/Agents/AgentBase.cs

```diff
@@ -197,6 +197,13 @@ static async ((AgentBase Agent, RpcRequest Request, TaskCompletionSource<RpcResp
         return await completion.Task.ConfigureAwait(false);
     }
 
+    public async ValueTask Publish<T>(T item, string? source = null) where T : IMessage
```

Comment:
this was overdue, thanks. Should it be called PublishMessage()?

Reply:
done